components/pipeline-service/production/base/pipelines-as-code-secret.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: pipelines-as-code-secret
+  namespace: openshift-pipelines
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  dataFrom:
+    - extract:
+        key: production/pipeline-service/github-app
+  refreshInterval: 1h
```

Comment:
For production we should probably be a bit more aggressive on the refreshes. I would much prefer 5mins in case secrets needs to be rotated (which I just had to do a month ago).

Reply:
This comment is not a blocker.